pkg/monitor/dynkeepalived.go

```diff
@@ -98,6 +98,27 @@ func KeepalivedWatch(kubeconfigPath, clusterConfigPath, templatePath, cfgPath st
 	var appliedConfig, curConfig, prevConfig *config.Node
 	var configChangeCtr uint8 = 0
 
+	// Lease VIPS
+	// To avoid backwards compatability - Bad flow doesn't return an error
+	if clusterName, _, err := config.GetClusterNameAndDomain(kubeconfigPath, clusterConfigPath); err == nil {
+		if exists, err := needLease(cfgPath); err == nil && exists {
+			vips := []VIP{{"api", apiVip}, {"ingress", ingressVip}}
+			if err = leaseVIPs(cfgPath, clusterName, vips); err == nil {
```

Comment:
In case that the leaseVIPs fails, what is the meaning of it in terms of successful completion of the installation?  Maybe VIPs are not available and we will not indicate it, only in the logs.

Reply:
Fixed.